.gitmodules

```diff
@@ -43,3 +43,6 @@
 [submodule "zenoh-kotlin"]
 	path = zenoh-kotlin
 	url = https://github.com/eclipse-zenoh/zenoh-kotlin.git
+[submodule "conan-recipes"]
```

Comment:
I fixed the filename. But the submodule still doesn't contain the zenoh-cpp recipe. And we're gonna need the zenoh-pico one too :/